contribs/ev/src/main/java/org/matsim/contrib/ev/discharging/DriveDischargingHandler.java

```diff
@@ -77,9 +78,10 @@ private boolean isOnFirstLink() {
 
 	private final Queue<LinkLeaveEvent> linkLeaveEvents = new ConcurrentLinkedQueue<>();
 	private final Queue<VehicleLeavesTrafficEvent> trafficLeaveEvents = new ConcurrentLinkedQueue<>();
-
+	private final QSim qsim;
```

Comment:
It would be nice to have a binding for the mobsim timer, so we do not need to inject the whole qsim. But that's something beyond this PR.